### DISCUSSION TITLE
What is the purpose of the --localize flag when packaging a solution?

### DISCUSSION THREAD

**Discussion by szmarci** [microsoft/powerplatform-build-tools#795] [Q&A]
According to the docs when using the `--localize` flag with `pac solution unpack`, it supposed to do this:

> Extract or merge all string resources into .resx files.

Which it does. It creates a Resources folder with a bunch of `.resx` files.

My understanding is that at this point I am free to modify these generated files, changing the translations, etc... And when I repackage the solution with this command `pac solution pack --localize` it reads these files during the process and puts them into the final `.xml`.

The problem is that this is not happening. I get these logs during packaging:

```
Found 47 localization resource(s) files
Localizing Component: SolutionManifest
Localizing Component: Entities
Localizing Component: Roles
Localizing Component: Workflows
Localizing Component: FieldSecurityProfiles
Localizing Component: Templates
Localizing Component: EntityMaps
Localizing Component: EntityRelationships
Localizing Component: optionsets
Localizing Component: MobileOfflineProfiles
Localizing Component: Dashboards
Localizing Component: InteractionCentricDashboards
Localizing Component: WebResources
Localizing Component: CustomControls
Localizing Component: AppModuleSiteMaps
Localizing Component: AppModules
Localizing Component: EntityDataProviders
Localizing Component: CanvasApps
Localizing 356 solution component files(s)...
```

But at the end of the process when I unzip the solution and try to look for my custom strings in `customizations.xml` I added before the packaging, they are nowhere to be found.

Is something I am misunderstanding about the purpose of this command?
Or is this a bug?

**Comment by MassoudDehkordi** [comment]
The issue @szmarci reported more than a year ago, is still present.
**pac** still cannot be used to replace **SolutionPackager** (something that we want to do) as L10N .resx files are not packed back into the XMLs.
